### PR TITLE
CLD-393: Document any difference backup/recovery and upgrades procedure in the ML Docker Func Spec 

### PR DESCRIPTION
**Pull request by barkhachoithani**
added steps to backup/restore a db

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -438,6 +438,22 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
...
+
+# Backing Up and Restoring a Database
+
+When creating a backup for a database on a MarkLogic Docker container, verify that the directory used for the backup is mounted to a directory on the Docker host machine or Docker volume. This is so that the database backup persists even after the container is stopped.
+
+This command is an example of mounting the directory /space used for backup on the Docker host file system, while running the MarkLogic Docker container.
+
+$ docker run -d -it -p 7000:8000 -p 7001:8001 -p 7002:8002 \
+     -v /data/MarkLogic:/var/opt/MarkLogic \
+     -v /space:/space \
```
Comment (vitalykorolev) [inline README.md:451]: Maybe it's best to use named volumes as we do in other examples as this is the best practice.
`--mount src=MarkLogic_backup_vol,dst=/space \`

File: README.md [hunk 2]
```diff
@@ -438,6 +438,22 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
...
+     -e MARKLOGIC_INIT=true \
+     -e MARKLOGIC_ADMIN_USERNAME=admin \
```
Comment (vitalykorolev) [inline README.md:453]: We should not have filled value of admin credentials especially something like admin:admin

File: README.md [hunk 3]
```diff
@@ -438,6 +438,22 @@ $ docker run -d -it -p 7200:8000 -p 7201:8001 -p 7202:8002 \
...
+     -e MARKLOGIC_ADMIN_PASSWORD=admin \
+     ml-docker-dev.marklogic.com/marklogic/marklogic-server-centos:10.0-8.3-centos-1.0.0-ea3
```
Comment (vitalykorolev) [inline README.md:455]: Better to use reference to our Docker hub image.

File: README.md (hunk 2 above)
Comment (barkhachoithani) [reply README.md:453]: right, I will update the credentials.

File: README.md (hunk 1 above)
Comment (barkhachoithani) [reply README.md:451]: will update the command to use named volume

File: README.md (hunk 3 above)
Comment (barkhachoithani) [reply README.md]: Updated to use the docker hub image. For now, just added the assumed image name for docker GA. 
Will update the image name before merging to the dev branch.

**Review by jmckean17 — APPROVED** [review]
This doesn't look right to me -  l-cluster-network'
I think it is supposed to be 'ml-cluster-network'

Everything else looks fine.

**Review by vitalykorolev — APPROVED** [review]

**Pull request merged**